plugins/ipc/loader.py

```diff
@@ -26,12 +26,22 @@ def load_all_extra_ipc_patients():
             Q_GET_DISTINCT_MRNS_FOR_TEST,
             params={'test_name': test.TEST_NAME}
         )
-        all_tested_mrns.update([r['Patient_Number'] for r in tested_mrns])
-
-    all_mrns = set(
+        # elcid does not use zero prefixed MRNs but the lab test
```

Comment:
This should probably be signalled harder, but this whole function is more of a development/testing thing, related to the initial development of `ipc.models.InfectionAlert` - that's paused for now, I think we'd just 'do this differently'. 
We should probably delete the whole function and anywhere it's called rather than add complexity

Reply:
cool, in that case we aren't using anything in plugins.ipc.loader so I have deleted it.